packages/components/src/components/tables/es-table/es-table.tsx

```diff
@@ -183,12 +183,19 @@ export class Table {
                         })}
                     >
                         {this.loading &&
-                            cell.loading?.(h, {
-                                ...(this.extraCellProps?.(key, data) ?? {}),
-                                key,
-                                data,
-                                parent: this.identifier,
-                            })}
+                            cell.width !== '0' &&
+                            (cell.loading === undefined ? (
+                                <es-es-loading-text
+                                    exectedLength={cell.title?.length ?? 30}
+                                />
+                            ) : (
+                                cell.loading &&
+                                cell.loading(h, {
+                                    key,
+                                    data,
+                                    parent: this.identifier,
+                                })
+                            ))}
```

Comment:
This entire section section has gotten a bit hard to read, probably time to split it out:

```tsx
    renderCell = (
        cell: TableCell<unknown>,
        name: string,
        key: string,
        data: unknown,
    ) => {
        if (this.loading) {
            if (cell.loading == null) {
                return (
                    <es-loading-text
                        expectedLength={cell.title?.length ?? 30}
                    />
                );
            }

            return cell.loading(h, {
                key,
                data,
                parent: this.identifier,
            });
        }

        if (cell.cell == null) {
            return autoExtract(data, name);
        }

        return cell.cell(h, {
            ...(this.extraCellProps?.(key, data) ?? {}),
            key,
            data,
            parent: this.identifier,
        });
    };
```